src/components/Transaction.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react'
 
-const Transaction = () => {
+const Transaction = (props) => {
   return (
```

Comment:
if you like, you can pull the transaction off of `props` here to avoid that repetition. `const transaction = props.transaction` or the shorthand `const {transaction} = props`